enterprise/models.py

```diff
@@ -61,6 +62,8 @@ class Meta(object):
     active = models.BooleanField(default=True)
     history = HistoricalRecords()
 
+    site = models.ForeignKey(Site, related_name="enterprise_customers")
```

Comment:
@e-kolpakov `NULL, in the database, is default to False. Currently I cannot create a customer without specifying the `site`. 

`